src/client/app/service/utils/utils.ts

```diff
@@ -0,0 +1,9 @@
+import {Injectable} from '@angular/core';
+
+@Injectable()
```

Comment:
I don't think you're injecting this class via DI at all; you should be able to remove this decorator.